maxfw/utils/image_functions.py

```diff
@@ -35,7 +35,8 @@ def to_pil_image(pic, mode=None):
     Returns:
         PIL Image: Image converted to PIL Image.
     """
-    if not(isinstance(pic, np.ndarray) or isinstance(pic, io.BytesIO)):
+    if not isinstance(pic, (bytes, bytearray)) and not(isinstance(pic, np.ndarray)):
```

Comment:
An image in bytes-format is not of the `io.BytesIO` type, but of the `bytes` or `bytearray` format.

Reply:
Should we convert it here or convert it in the flask API?